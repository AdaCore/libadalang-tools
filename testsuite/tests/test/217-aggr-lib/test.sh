#!/bin/bash

gnattest -P root.gpr --stub --stubs-dir ../stubs --tests-dir ../tests -q
gprbuild -P obj/gnattest_stub/harness/test_drivers.gpr -q
gnattest obj/gnattest_stub/harness/test_drivers.list
