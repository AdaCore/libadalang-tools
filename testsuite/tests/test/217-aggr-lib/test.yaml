description: Check that the stubbing mode of gnattest correctly supports
  aggregate library projects.

driver: shell_script
