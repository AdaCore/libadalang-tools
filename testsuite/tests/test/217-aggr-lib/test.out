pkg_root.ads:3:4: info: corresponding test PASSED
pkg1.ads:3:4: error: corresponding test FAILED: Test not implemented. (pkg1-test_data-tests.adb:44)
pkg2.ads:3:4: error: corresponding test FAILED: Test not implemented. (pkg2-test_data-tests.adb:44)
3 tests run: 1 passed; 2 failed; 0 crashed.
